.gitignore

```diff
@@ -4,3 +4,4 @@ vendor/
 composer.lock
 Modules/
 build/
+nbproject/
```

Comment:
This seems to be something related to your environment,  please add this to your global .gitignore file instead.